### ISSUE TITLE
Updated signatures repository

### ISSUE THREAD

**Issue by jacoporusso-css** [cert-ee/cuckoo3#78]
Setting up a fresh instance, I noticed signatures.zip was last updated 2 years ago.
After some investigation (due to lack of documentation) here's what I found:

- unlike cuckoo original project, there's no community repository (https://github.com/cuckoosandbox/community)
- rules are written in yml format, but there's no definition of the allowed fields that can be used in building a custom signature
- there's plenty of new rules in other projects (e.g.: https://github.com/CAPESandbox/community/tree/master/modules/signatures), but since they're all written in python, there's no immediate way to convert them to desired yml format. Is there a supported way to do it?

Thanks

**Comment by Cryss76** [comment]
yeah there isn't a community-repo but maybe this can help you: https://github.com/cert-ee/cuckoo3_signatures
Does are the used signatures in the zip-file

There are indeed no definition but I was able to figure the gist of it. The start is pretty boring as it is just naming and self explanatory stuff.

Where stuff gets interesting are the triggers. The triggers are as far as I understand mappings to threemons events and compose out of an event kind and action/status. For example "file write" : file is the event kind and action is write.
The problem is where to find what event kinds and actions/status can be:
Event kinds can be found [here](https://github.com/cert-ee/cuckoo3/blob/13cbd126c35226337bd47c3b16ddc03f3790d37c/processing/cuckoo/processing/event/events.py#L6).
On the same file (processing/cuckoo/processing/event/events.py) you can find the aktions/status but those are ambiguous because of the multiple definitions. The dictionaries that end with simplified are always the right aktion/status. (example [here](https://github.com/cert-ee/cuckoo3/blob/13cbd126c35226337bd47c3b16ddc03f3790d37c/processing/cuckoo/processing/event/events.py#L56)).
If there is no *simplified dict use the strings in class eventAktion or eventStatus like in [class ProcessStatuses](https://github.com/cert-ee/cuckoo3/blob/13cbd126c35226337bd47c3b16ddc03f3790d37c/processing/cuckoo/processing/event/events.py#L110C7-L110C22). Those the defined the aktions/status that cuckoo3 uses.

**Comment by rkivistik** [comment]
As @Cryss76 pointed out, https://github.com/cert-ee/cuckoo3_signatures is our community repo. Unfortunetly the repo README does not reflect that.
And thank you describing some of the process. At my current knowledge, there is no guide per se to explain the process. I will look into it to see if one can be made.

Once we can get Cuckoo3 to a more suitable place in terms of development, we are hoping the community can join in in creating more signatures.
Stay tuned for our roadmap.

Im closing this issue as @Cryss76 has provided a nice explanation.
